### PR TITLE
Psp 4695

### PR DESCRIPTION
**Pull request by devinleighsmith**
add the edit property page to the acquisition file

### CODE REVIEW

File: source/frontend/src/features/properties/map/acquisition/ViewSelector.tsx [hunk 1]
```diff
@@ -33,7 +34,17 @@ export const ViewSelector = React.forwardRef<FormikProps<any>, IViewSelectorProp
...
 
         case EditFormNames.propertyDetails:
-          return <></>;
+          const propertyFile = getAcquisitionFileProperty(
+            props.acquisitionFile,
+            props.selectedMenuIndex,
+          );
+          return (
+            <UpdatePropertyDetailsContainer
+              id={propertyFile?.property?.id as number}
```
Comment (FuriousLlama) [inline source/frontend/src/features/properties/map/acquisition/ViewSelector.tsx:43]: nit, is the cohesion necessary?

File: source/frontend/src/features/properties/map/acquisition/ViewSelector.tsx [hunk 2]
```diff
@@ -48,27 +59,37 @@ export const ViewSelector = React.forwardRef<FormikProps<any>, IViewSelectorProp
           />
         );
       } else {
-        const properties = props.acquisitionFile?.fileProperties || [];
-        const selectedPropertyIndex = props.selectedMenuIndex - 1;
-        const acquisitionFileProperty = properties[selectedPropertyIndex];
-        acquisitionFileProperty.file = props.acquisitionFile;
-        return (
+        return !!props.acquisitionFile ? (
```
Comment (FuriousLlama) [inline source/frontend/src/features/properties/map/acquisition/ViewSelector.tsx:62]: nit an IF would be cleaner IMO

File: source/frontend/src/features/properties/map/acquisition/ViewSelector.tsx (hunk 1 above)
Comment (devinleighsmith) [reply source/frontend/src/features/properties/map/acquisition/ViewSelector.tsx:43]: I can probably refactor this to get rid of this cast, yes.

File: source/frontend/src/features/mapSideBar/MotiInventoryContainer.tsx [hunk 3]
```diff
@@ -26,9 +26,7 @@ export const MotiInventoryContainer: React.FunctionComponent<IMotiInventoryConta
 
   const [isEditing, setIsEditing] = useState<boolean>(false);
 
-  const [formikRef, setFormikRef] = useState<React.RefObject<FormikProps<any>> | undefined>(
-    undefined,
-  );
+  const formikRef = useRef<FormikProps<any>>(null);
```
Comment (asanchezr) [inline source/frontend/src/features/mapSideBar/MotiInventoryContainer.tsx:29]: thanks for fixing this

File: source/frontend/src/features/mapSideBar/tabs/propertyDetails/update/UpdatePropertyDetailsContainer.tsx [hunk 4]
```diff
@@ -4,7 +4,8 @@ import useIsMounted from 'hooks/useIsMounted';
 import { useQueryMapLayersByLocation } from 'hooks/useQueryMapLayersByLocation';
 import isNumber from 'lodash/isNumber';
 import { Api_Property } from 'models/api/Property';
-import React, { useEffect, useRef, useState } from 'react';
+import React from 'react';
+import { useEffect, useState } from 'react';
```
Comment (asanchezr) [inline source/frontend/src/features/mapSideBar/tabs/propertyDetails/update/UpdatePropertyDetailsContainer.tsx:8]: nit: why the duplicate import lines? was this auto-generated by the linter? 
I would expect all imports from same package/library to be on a single line...

**Review by asanchezr — APPROVED** [review]

**Pull request merged**